src/test/resources/oas/v3/OpenAPI-Specification/uspto/src/main/resources/api/uspto.yaml

```diff
@@ -0,0 +1,210 @@
+openapi: 3.0.1
+servers:
+  - url: '{scheme}://developer.uspto.gov/ds-api'
```

Comment:
uspto ???

Reply:
It's just a public API taken from the web. I wouldn't update it but I'll rename the file. Is that ok for you?